doc/lua_api.txt

```diff
@@ -2645,6 +2646,11 @@ Some types may inherit styles from parent types.
     * noclip - boolean, set to true to allow the element to exceed formspec bounds. Default to false.
 * label, vertlabel
     * noclip - boolean, set to true to allow the element to exceed formspec bounds.
+    * halign - left/center/right, sets text horizontal alignment in respect to position.
```

Comment:
Please split this into align and origin. Align should set `setTextAlignment`. Origin should be used for the positioning logic

Reply:
Renaming `align` to `origin` and then inverting would be fine with me.

I had no plans to make a separate `align` for multiple lines because I don't think it would be very useful (you don't know the width of the rect, so it makes it hard to know how to place it on the form), but I can add it if you want.

Reply:
> you don't know the width of the rect, so it makes it hard to know how to place it on the form, and plus, it might confuse people

The code already supports alignment though? `e->setTextAlignment(halign, valign);`

Reply:
> then inverting would be fine with me

What you currently call align shouldn't be inverted - that's correct.

--------------------

Text alignment only really makes sense with multi-line labels, and multi-line labels are strange with the label[] element